server/grpc/api/execution.go

```diff
@@ -24,7 +28,12 @@ func NewServer(sdk *sdk.SDK) *Server {
 
 // Get returns execution from given hash.
 func (s *Server) Get(ctx context.Context, req *api.GetExecutionRequest) (*definition.Execution, error) {
-	exec, err := s.sdk.GetExecution(req.Hash)
+	hash, err := hex.DecodeString(req.Hash)
```

Comment:
not sure why this is here? You changed the `GetExecutionRequest` hash to string so there is no decode needed

Reply:
yes it is, because execution.Execution uses bytes for store hash. grpc services are the only one using string for hashes so you need to decode it first.